contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -111,6 +111,8 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
         operator.stakingLimit = _stakingLimit;
 
         emit NodeOperatorAdded(id, _name, _rewardAddress, _stakingLimit);
+
+        return id;
```

Comment:
This is out of scope. Should be done in a separate issue and in separate commit.
I'd propose to drop this change.